INSTALL.rst

```diff
@@ -23,7 +41,9 @@ including `Anaconda <https://www.anaconda.com/download/>`_,
 
 On all major operating systems, install it via shell/command prompt::
 
-  pip install scikit-image
+  pip install --user scikit-image
```

Comment:
I'm not sure if this is correct for venv and conda envs

Reply:
You're right, I don't think this will do the right thing in a conda-env. I'm not sure about venv.

Reply:
Yeah, this is not the correct way to do it, this will install it in a systemwide user directory, better than encouraging `sudo pip` but not better than teaching about venv

```
$ ls /home/mark/.local/lib/python3.7/site-packages
cycler-0.10.0.dist-info                     numpy-1.17.4.dist-info
cycler.py                                   PIL
dateutil                                    Pillow-6.2.1.dist-info
decorator-4.4.1.dist-info                   __pycache__
decorator.py                                pylab.py
imageio                                     pyparsing-2.4.5.dist-info
imageio-2.6.1.dist-info                     pyparsing.py
kiwisolver-1.1.0.dist-info                  python_dateutil-2.8.1.dist-info
kiwisolver.cpython-37m-x86_64-linux-gnu.so  PyWavelets-1.1.1.dist-info
matplotlib                                  pywt
matplotlib-3.1.1.dist-info                  scikit_image-0.16.2.dist-info
matplotlib-3.1.1-py3.7-nspkg.pth            scipy
mpl_toolkits                                scipy-1.3.2.dist-info
networkx                                    six-1.13.0.dist-info
networkx-2.4.dist-info                      six.py
numpy                                       skimage
```

Reply:
this was from within a conda env.

Reply:
I guess this advice needs to be tested reasonably thoroughly:

- what happens when pip is system pip
- what happens when a venv is activated
- what happens when a conda-env is activated?

Reply:
yeah, at this stage we need to rework the whole install procedure... working on that.

Reply:
> what happens when pip is system pip

On my system (Manjaro), `pip install` will fail without using either `sudo` or `--user` with a "permission denied" error. Likely the same for other Linux-based OSs?

> what happens when a conda-env is activated

As far as I know, conda installs pip automatically in any environment with Python. Activating that environment will also point `pip` at the environment version. Using pip from within a conda environment works good for pure-Python packages (and dependencies) but I've frequently encountered compatibility issues when mixing binary packages from PyPI and conda's repos (doesn't help that conda ships a [very old version of `ld`](https://github.com/ContinuumIO/anaconda-issues/issues/11152)).

Reply:
I know, this whole thing of installing software is a mess. The goal of this PR is mostly to avoid people going:

1. I tried `pip install scikit-image` it said permission error.
2. I gave it permissions `sudo !!` because my friend told me that `sudo !!` is a cool shotcut.
3. Now my whole system is broken.

Even Ubuntu 18.04 ships with a version of numpy much too old for scikit-image 0.16

https://packages.ubuntu.com/bionic/python/python3-numpy